packages/block-library/src/embed/wp-embed-preview.js

```diff
@@ -18,6 +20,36 @@ class WpEmbedPreview extends Component {
 		this.node = createRef();
 	}
 
+	componentDidMount() {
+		window.addEventListener( 'message', this.resizeWPembeds );
+	}
+
+	componentWillUnmount() {
+		window.removeEventListener( 'message', this.resizeWPembeds );
+	}
+
+	/**
+	 * Checks for WordPress embed event signaling the height
+	 * change when content loads.
+	 *
+	 * @param {WPSyntheticEvent} event Message event.
+	 */
+	resizeWPembeds( { data: { secret, message, value } = {} } ) {
+		if ( [ secret, message, value ].some( ( attribute ) => ! attribute ) )
+			return;
+
+		if ( message === 'height' ) {
+			const iframes = document.querySelectorAll(
+				`iframe[data-secret="${ secret }"`
+			);
+			( iframes || [] ).forEach( ( iframe ) => {
```

Comment:
```suggestion
			iframes.forEach( ( iframe ) => {
```

Doesn't seem like `querySelectorAll` ever returns anything other than a NodeList.